.gitignore

```diff
@@ -124,3 +124,4 @@ package-lock.json
 
 # ignore renamed paths
 packages/nimble-blazor
+/angular-workspace
```

Comment:
Only other non-mechanical change. Ignoring the renamed path to ease transition for those switching between branches (any cruft like generated icons in old folders don't show up as local changes when sinking down)